server/celery_backend/tasks/upload_feedback_dump.py

```diff
@@ -0,0 +1,47 @@
+import csv
+import io
+from datetime import datetime
+
+from celery_backend.tasks.database import AppDatabase, LogDatastore
+from module.services.model.feedback import Feedback
+
+from ..celery_app import app
+from . import constants
+
+feedback_store = LogDatastore()
+
+
+@app.task(name="upload.feedback.dump")
+def upload_feedback_dump() -> None:
+    """Uploads feedback dumps to cloud storage"""
+    file = io.StringIO()
+    csv_writer = csv.writer(file)
+    app_db = AppDatabase()
+
+    collection = app_db.get_collection("feedback")
+
+    d = datetime.now()
+    start_date = d.replace(
+        month=d.month - 1, day=1, hour=0, minute=0, second=0, microsecond=0
+    ).timestamp()
+    end_date = d.replace(
+        month=d.month, day=1, hour=0, minute=0, second=0, microsecond=0
+    ).timestamp()
+    query = {
+        "feedbackTimeStamp": {"$gte": int(start_date), "$lt": int(end_date)},
+    }
+
+    for doc in collection.find(query):
+        feedback = Feedback(**doc)
+        csv_writer.writerow(feedback.to_export_row())
```

Comment:
Before writing the rows, shall we write a header mentioning the column names?

Reply:
Done